recipes/syft-proto/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "syft-proto" %}
+{% set version = "0.4.3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: fa1760b714fb035a8b42d8d3d51ac12a361fa93f99ef8937d2bc5d021f152309
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - protobuf >=3.11.1
+    - python
+
+test:
+  imports:
+    - syft_proto
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/syft-proto/
+  summary: PySyft protocol constants.
+  license: Apache
+  license_file: LICENSE
```

Comment:
Since the license is being manually packaged, please add a link above the `license_file` line with a link to:

https://github.com/OpenMined/syft-proto/pull/103

as a way of tracking and as a reminder to check if the license is included in the source distribution in a future release.

Reply:
your pr has been merged now, would be worth doing it now?

Reply:
I presume you would need to push out a new release to pypi first, but if you want to do that, you could base the recipe off of the new version.

Reply:
A few other notes:

- It looks like the current version on pypi is 0.4.4
- I think the upstream packaging is broken since the `setup.py` copies `proto.json`, but that file isn't in the source distribution, which is causing the build here to fail.

Reply:
already opened a pr to add that file
https://github.com/OpenMined/syft-proto/pull/102